presentation/backend/src/presentation-backend/PresentationManager.ts

```diff
@@ -371,14 +388,8 @@ export class PresentationManager {
    * @public
    */
   public async getNodes(requestOptions: WithCancelEvent<Prioritized<Paged<HierarchyRequestOptions<IModelDb, NodeKey, RulesetVariable>>>> & BackendDiagnosticsAttribute): Promise<Node[]> {
-    const { rulesetOrId, parentKey, ...strippedOptions } = requestOptions;
-    const params = {
-      requestId: parentKey ? NativePlatformRequestTypes.GetChildren : NativePlatformRequestTypes.GetRootNodes,
-      rulesetId: this._detail.registerRuleset(rulesetOrId),
-      ...strippedOptions,
-      nodeKey: parentKey,
-    };
-    return JSON.parse(await this._detail.request(params), Node.listReviver);
+    const nodes = await this._detail.getNodes(requestOptions);
```

Comment:
I think it would be a good idea to make similar change to all functions, e.g. `getNodesCount`. Currently we're incosistent - in some cases `params` are created in the *manager*, in some others they're created in the *detail*.